FirebaseFunctions.podspec

```diff
@@ -54,8 +54,19 @@ Cloud Functions for Firebase.
   s.test_spec 'integration' do |int_tests|
     int_tests.scheme = { :code_coverage => true }
     int_tests.source_files = 'FirebaseFunctions/Tests/Integration/*.[mh]',
-                             'FirebaseFunctions/Tests/Integration/*.plist',
                              'SharedTestUtilities/FIRAuthInteropFake*',
                              'SharedTestUtilities/FIRMessagingInteropFake*'
   end
+
```

Comment:
Strange that this change is in the diff. Perhaps you didn't start the PR from the latest master?

Reply:
So I noticed that the `FirebaseFunctions.podspec` was the only `podspec` that changed since I staged them last week. 

So to get the diff seen here, I checked out the `FirebaseFunctions.podspec` from latest master and manually changed the version to 9.10.0. 

Was this the correct workflow for this scenario (podspec file changes after staging it on specs staging)?

Reply:
This makes sense. I didn't realize you started last week.